pkg/yttlibrary/overlay/match_annotation_expects_kwarg.go

```diff
@@ -115,18 +119,29 @@ func (a MatchAnnotationExpectsKwarg) checkString(typedVal starlark.String, num i
 	return fmt.Errorf("Expected '%s' to be in format 'i+' where i is an integer", typedValStr)
 }
 
-func (a MatchAnnotationExpectsKwarg) checkList(typedVal *starlark.List, num int) error {
+func (a MatchAnnotationExpectsKwarg) checkList(typedVal *starlark.List, matches []*filepos.Position) error {
 	var lastErr error
 	var val starlark.Value
 
 	iter := typedVal.Iterate()
 	defer iter.Done()
 
 	for iter.Next(&val) {
-		lastErr = a.checkValue(val, num)
+		lastErr = a.checkValue(val, matches)
 		if lastErr == nil {
 			return nil
 		}
 	}
 	return lastErr
 }
+
+func formatPositions(pos []*filepos.Position) string {
```

Comment:
lets place formatPositions on MatchAnnotationExpectsKwarg. you can use `func (MatchAnnotationExpectsKwarg) formatPositions(...` sig with naming `MatchAnnotationExpectsKwarg`.

Reply:
Yeah, not sure why I didn't do that: it's an implementation detail of `#Check()`.